igvc_description/launch/spawn_swervi.launch

```diff
@@ -15,7 +15,7 @@
     <!-- End of arguments list -->
 
     <!-- Spawn the vehicle. -->
-    <param name="robot_description" command="$(find xacro)/xacro $(arg model) gpu:=$(arg gpu)" />
```

Comment:
Does this need to be changed? Isn't the default argument the same as the proposed change?

Reply:
actually this is fine